package.json

```diff
@@ -133,6 +133,7 @@
 		"bunyan": "^1.8.12",
 		"busboy": "^0.2.14",
 		"cas": "https://github.com/kcbanner/node-cas/tarball/fcd27dad333223b3b75a048bce27973fb3ca0f62",
+		"cld": "^2.4.8",
```

Comment:
I think this dependency has to be located in the package `assistify:dbs-translation` itself (see https://docs.meteor.com/api/packagejs.html#PackageNpm-depends).